subsys/bluetooth/host/att.c

```diff
@@ -2285,3 +2285,24 @@ void bt_att_req_cancel(struct bt_conn *conn, struct bt_att_req *req)
 
 	att_req_destroy(req);
 }
+
+u16_t bt_att_get_payload(struct bt_conn *conn, uint8_t op)
```

Comment:
I know the name gets rather long, but this should really be get_payload_len(), since it doesn't return the payload, rather its length.

Reply:
Sure, I will adjust that.